exercises/saddle-points/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+You are planning on building a tree house in the woods near your house so that you can watch the sun rise and set.
+
+You've obtained data from a local survey company that shows the heights of all the trees in each rectangular section of the map.
+
+You need to analyze each grid on the map to find the perfect tree for your tree house.
+
+The best tree will be the tallest tree compared to all the other trees to the east and west, so that you have the best possible view of the sunrises and sunsets.
+
+You don't like climbing too much, so the perfect tree will also be the shortest among all the trees to the north and to the south.
```

Comment:
I suggest not having each sentence as a paragraph here. Maybe L5+7 could be merged into one paragraph. And 9+11?